packages/driver/src/cy/net-stubbing/route-matcher-log.ts

```diff
@@ -0,0 +1,72 @@
+import _ from 'lodash'
+
+import {
+  RouteMatcherOptions,
+  StaticResponse,
+} from '@packages/net-stubbing/lib/types'
+import $utils from '../../cypress/utils'
+
+// properties that, if they are the only ones set, make sense to display as the display url
+const standaloneDisplayUrlProps: Array<keyof RouteMatcherOptions> = ['url', 'path', 'pathname']
+
+export function _getDisplayUrlMatcher (matcher: RouteMatcherOptions): string {
```

Comment:
Prefixing a function name with an underscore should indicate that it's a private function. I'd suggest removing the underscore, since an exported function is not private.

Reply:
I've seen the underscore notation used elsewhere in our codebase to denote something that is only exported for use in tests, but I guess not in the `driver` as much. Updated.

Reply:
I've certainly done that before, but I think the distinction is that the function needs to not just be exported only for the test, but it also should only be _used_ in the test. Since this function is actually utilized outside of a test, it doesn't meet that criteria.

For example, I know I have added a `__reset` method to a singleton so that its state can be reset or to reset global state, since there's no other way to accomplish it, but it's only used in the test and not in the standard execution of the code. So it's basically using an anti-pattern to compensate for an anti-pattern 😅. With newer code, I think we're better about not creating singletons or global state, so hopefully it's not as necessary.